### DISCUSSION TITLE
game crashed after switching to fullscreen, doesn't start anymore

### DISCUSSION THREAD

**Discussion by jojopipe** [ppy/osu#18525] [Q&A]
### Type

Crash to desktop

### Bug description

When I wanted to disable mouse buttons during gameplay via F10 I accidentally hit F11, so the game went into windowed mode. I immediately hit F11 again to return to fullscreen mode, which crashed the game. Now when I try to open the game, it only shows a blackscreen for 2 seconds and closes again.

### Screenshots or videos

blackscreen and crash on start:
https://user-images.githubusercontent.com/78083515/170712578-b05858c6-c655-4dad-874b-c6a190fb22ae.mp4

unfortunately no video of the initial crash when I went into fullscreen mode.

### Version

2022.523.0 (newest)

### Logs

[updater.log](https://github.com/ppy/osu/files/8787078/updater.log)
[network.log](https://github.com/ppy/osu/files/8787080/network.log)
[performance.log](https://github.com/ppy/osu/files/8787081/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/8787082/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/8787083/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/8787084/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/8787085/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/8787086/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8787088/updater.log)

**Comment by jojopipe** [comment]
On Windows 11, I should probably add :)

**Comment by bdach** [comment]
The logs cut out abruptly. To be able to diagnose this further, please attach Windows Event Viewer info by performing the following steps:
1. Type `Event viewer` in the Windows search bar (shortcut: <kbd>Windows</kbd>+<kbd>S</kbd>) and press <kbd>Enter</kbd>, or open the *Run...* button (<kbd>Windows</kbd>+<kbd>R</kbd>), type in `eventvwr` and press <kbd>Enter</kbd>.
2. The *Event viewer* window will appear. Expand *Windows Logs* on the left and choose *Application*. It might take a while to load.
3. In the *Actions* pane on the right click *Save All Events As...*, then choose a location to save to.
4. On the *Display Information* popup, select the second option, then the checkbox to show all available languages, then *English (UK)* (US would also do fine).
5. Append the resulting `.evtx` file to this issue.

**Comment by yumetsuro** [comment]
Hi, i have the same issue: When i open the game and I'm Alt-Tabbed away from it it's work, but as soon as i click in the game windows the game freezes and close after few seconds. I'm on Windows 11 as well
Unfortunately i can't append here a file with `.evtx` extension
But the module that gave the error is here `AppData\Local\osulazer\app-2022.509.0\coreclr.dll`

**Comment by Sublimelime** [comment]
You can place the evtx file inside of a zip, that'll let you upload it.

**Comment by yumetsuro** [comment]
Just tried to delete that file and reinstall the game, I also updated my drivers and the game started without crashing.
So if one of that worked other people can try as well doing that

**Comment by bdach** [comment]
@U-n-Own please do NOT remove game files like that. if anything of what you listed above helped, it was most likely the driver update.
